_gsocblogs/2022/blog_Rucio_NimishBongale.md

```diff
@@ -0,0 +1,127 @@
+---
+project: HSF Rucio WebUI
+title: Rucio WebUI Midterm Progress Report 
+author: Nimish Bongale
+photo: blog_authors/NimishBongale.jpeg
+date: 18.08.2022 
+year: 2022
+layout: blog_post
+logo: RUCIO-logo.jpg
+intro: Let’s have a look at my progress, learning, and overall experience during the GSoC 2022 period, shall we?
+---
+
+## Project Synopsis
+
+### Mentors: Mayank Sharma, Martin B, Mario Lassnig
+### Project Proposal link: [https://summerofcode.withgoogle.com/proposals/details/U9JdPuHt](https://summerofcode.withgoogle.com/proposals/details/U9JdPuHt)
```

Comment:
This link gives a `restricted access` error, please modify it to something public.

Reply:
Done! 

This link should now be accessible:

https://summerofcode.withgoogle.com/programs/2022/projects/1ps6wuOz